### PR TITLE
add more ICS modules

### PR DESCRIPTION
**Pull request by Lextum**

### CODE REVIEW

File: data/modules_i.json [hunk 1]
```diff
@@ -24,32 +26,36 @@
...
     "DEFRC1.2": "Zusatzmodul",
     "DEFRC1_1": "Zusatzmodul",
     "DESG": "Zusatzmodul",
+    "DIBM": "Erweiterungsmodul",
     "DIGITRANS": "Zusatzmodul",
+    "DITI": "Erweiterungsmodul",
     "DL4G": "Majormodul",
     "DMATH": "Kernmodul",
-    "EAEK": "Erweiterungsmodul",
-    "EAUS": "Erweiterungsmodul",
```
Comment (eddex) [inline data/modules_i.json]: have those modules been removed from the i modules website?

File: tools/parse_module_type.py [hunk 2]
```diff
@@ -80,11 +80,28 @@ def parseWebsite():
...
+    modules_with_type['ANLS'] = kernmodul
+    modules_with_type['SPTA'] = kernmodul
+    modules_with_type['CRS'] = erweiterungsmodul
+    modules_with_type['DSO'] = kernmodul
+    modules_with_type['PRIVACY1'] = kernmodul
+    modules_with_type['OSA'] = kernmodul
+    modules_with_type['ISM'] = kernmodul
+
+    # only for ics students
+    # modules_with_type['ETHIK'] = kernmodul
```
Comment (eddex) [inline tools/parse_module_type.py]: there might be more modules like this.. see #9. We can leave the commented out line in like it is for now. But issue #9 should be addressed to fix this.

File: data/modules_i.json (hunk 1 above)
Comment (Lextum) [reply data/modules_i.json]: yes, can't find them anymore.

File: tools/parse_module_type.py (hunk 2 above)
Comment (Lextum) [reply tools/parse_module_type.py]: I haven't checked all I and WI modules, but I will add them for the moment like this, until #9 is implemented.

Comment (Lextum) [reply tools/parse_module_type.py]: Or HSLU updates the ICS module page and the type can be extracted automatically

**Pull request merged**